__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
```

Comment:
Don't necessarily need to destructure topicData, userData etc. can just have it as data - can leave as is for now :)